net/libfab/libfab.c

```diff
@@ -662,8 +695,24 @@ static int libfab_ep_find(struct m0_net_transfer_mc *tm, const char *name,
 		if (name != NULL)
 			rc = libfab_ep_create(tm, name, epn, epp);
 		else {
-			sprintf(ep, "libfab:%s:%s", epn->fen_addr, epn->fen_port);
-			rc = libfab_ep_create(tm, ep, epn, epp);
+			sprintf(ep_str, "libfab:%s:%s", epn->fen_addr,
+				epn->fen_port);
+			rc = libfab_ep_create(tm, ep_str, epn, epp);
+		}
+	} else {
+		if (name != NULL && epn != NULL) {
```

Comment:
From this checking, I can see that "epn" might be NULL.
So, it is necessary to checking if it is NULL on line 698.

Reply:
Added M0_ASSERT(epn != NULL)
In this function either one of the arguments from name or epn must not be NULL.